christfellowshipchurch/apollos-api
# Add contactGroupLeader mutation

## DESCRIPTION
Adds `contactGroupLeader` to trigger workflow to contact a group leader in the group finder.

### 1. How do I test this PR?
* Fire up `web-app-v2` on the `connect-workflow` branch
*  Navigate to the group finder results page
![image](https://user-images.githubusercontent.com/2528817/107540954-398bf880-6b8c-11eb-99c8-44837880bb23.png)
* Click on contact on one of the groups
* Send a message
![image](https://user-images.githubusercontent.com/2528817/107540338-90dd9900-6b8b-11eb-924c-5faa0e1f1cd6.png)

* Verify success modal shows up
* Go to Rock `workflowTypeId=654` make sure you see a recent entry from apollos REST

![image](https://user-images.githubusercontent.com/2528817/107540509-be2a4700-6b8b-11eb-9f5d-d499b124647f.png)

## TODO:

- [x] No new warnings in tests, storybook, or apps
- [x] Assign relevant reviewers
- [x] Upload images Web
- [ ] Get one peer approval before merging

## REVIEW:

**Manual QA**

- [ ] QA branch on iOS and ensure it looks/behaves as expected
- [ ] QA branch on Android and ensure it looks/behaves as expected
- [ ] QA branch on Web and ensure it looks/behaves as expected

**Code Review**


> The purpose of PR Review is to _improve the quality of the software._
